src/endless/EndlessUsbToolDlg.cpp

```diff
@@ -2692,29 +2692,29 @@ HRESULT CEndlessUsbToolDlg::OnSelectFileNextClicked(IHTMLElement* pElement)
 	}
 
     // Get display name with actual image size, not compressed
-    CString selectedImage, personality, version, date, selectedSize;
+    CString selectedImage, personality, version;
     ULONGLONG size = 0;
-    bool isInstallerImage = false;
+
     if (m_useLocalFile) {
         selectedImage = UTF8ToCString(image_path);
         pFileImageEntry_t localEntry = NULL;
         if (!m_imageFiles.Lookup(selectedImage, localEntry)) {
             uprintf("ERROR: Selected local file not found.");
-        } else {
-            size = m_selectedInstallMethod == InstallMethod_t::TryEndless && !localEntry->isUnpackedImage ? GetExtractedSize(selectedImage, FALSE) : localEntry->size;
-
-			if (localEntry->isUnpackedImage) {
-				selectedImage = CSTRING_GET_PATH(localEntry->unpackedImgSigPath, '.');
-			}
         }
-        selectedImage = CSTRING_GET_LAST(selectedImage, '\\');
+
+		personality = localEntry->personality;
```

Comment:
This will crash in the error path. I think rather than leaning on dereferencing NULL to crash the process, we should advance to the error page if `.Lookup()` returns false.